test/unit/ui/handler/touch_zoom_rotate.test.js

```diff
@@ -40,8 +40,9 @@ test('TouchZoomRotateHandler fires zoomstart, zoom, and zoomend events at approp
 
     simulate.touchend(map.getCanvas(), {touches: []});
     map._renderTaskQueue.run();
-    t.equal(zoomstart.callCount, 1);
-    t.equal(zoom.callCount, 2);
+    // incremented because inertia starts a second zoom
```

Comment:
:thinking: I'm not sure I understand what's going on here, why would this change affect this test? Seems like maybe something we need to fix in the `TouchZoomRotateHandler`?

Reply:
Yeah, it took me a while to figure this out. A touch zoom can trigger two separate zoom events: one for while the fingers are down and one for the inertia zoom afterwards. Previously this wasn't showing up in the test because the map's minzoom was preventing the inertia zoom. The updated tests reflect the existing, actual behavior.

https://github.com/mapbox/mapbox-gl-js/blob/4e12ca76c45ef3923456f6245fb4565674fdf44f/src/ui/handler/touch_zoom_rotate.js#L237
https://github.com/mapbox/mapbox-gl-js/blob/4e12ca76c45ef3923456f6245fb4565674fdf44f/src/ui/handler/touch_zoom_rotate.js#L280

Reply:
Oof - adding this to my long list of things to fix in the gesture handlers. OK for changing the test to reflect actual behavior, thanks for catching this

Reply:
Opened https://github.com/mapbox/mapbox-gl-js/issues/9034